docs/topics/tour/kotlin-tour-hello-world.md

```diff
@@ -0,0 +1,115 @@
+[//]: # (title: Hello world)
+
+<microformat>
+    <p>This is the first part of the Kotlin tour:</p>
+    <p><img src="icon-1.svg" width="20" alt="First step" /> <strong>Hello world</strong><br />
+        <img src="icon-2-todo.svg" width="20" alt="Second step" /> <a href="kotlin-tour-basic-types.md">Basic types</a><br />
+        <img src="icon-3-todo.svg" width="20" alt="Third step" /> <a href="kotlin-tour-collections.md">Collections</a><br />
+        <img src="icon-4-todo.svg" width="20" alt="Fourth step" /> <a href="kotlin-tour-control-flow.md">Control flow</a><br />
+        <img src="icon-5-todo.svg" width="20" alt="Fifth step" /> <a href="kotlin-tour-functions.md">Functions</a><br />
+        <img src="icon-6-todo.svg" width="20" alt="Sixth step" /> <a href="kotlin-tour-classes-part-1.md">Classes</a><br />
+        <img src="icon-7-todo.svg" width="20" alt="Final step" /> <a href="kotlin-tour-null-safety.md">Null safety</a></p>
+</microformat>
+
+Below is a simple program that prints "Hello, world!":
+
+```kotlin
+fun main() {
+    println("Hello, world!")
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="hello-world-kotlin"}
+
+In Kotlin:
+* `fun` is used to declare a function
+* the entry point is the `main` function
+* function arguments are written within parentheses `()`
+* the body of a function is written within curly braces `{}`
+* [`println()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/println.html) and [`print()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/print.html) functions print their arguments to standard output
+
+Functions are discussed in more detail in a couple of chapters. Until then, all examples use the `main` function.
+
+## Variables
+
+All programs need to be able to store data, and variables help you to do just that. In Kotlin, you can declare:
+* read-only variables with `val`
+* mutable variables with `var`
+
+To assign a value, use the assignment operator `=`.
+
+```kotlin
+fun main() { 
+//sampleStart
+    val popcorn = 5
+    val hotdog = 7
+    var customers = 10
+    
+    //Some customers leave the queue
+    customers = 8
+//sampleEnd
+}
```

Comment:
I was trying to keep this example as simple as possible so that types are only really introduced and discussed in the next chapter: Basic types. Is that OK?